qiskit/backends/ibmq/ibmqprovider.py

```diff
@@ -58,6 +59,14 @@ def deprecated_backend_names(self):
             'real': 'ibmqx1'
             }
 
+    def aliased_backend_names(self):
```

Comment:
Shouldn't Tokyo be here?

Reply:
Oh, nevermind.

Reply:
yeah tokyo is already correct in the API.